src/ProjectOrigin.Registry.Server/Startup.cs

```diff
@@ -34,44 +46,34 @@ public void ConfigureServices(IServiceCollection services)
             configuration.Bind(settings);
         });
 
+        services.AddOptions<VerifiableEventStoreOptions>().Configure<IConfiguration>((settings, configuration) =>
+        {
+            configuration.GetSection("VerifiableEventStore").Bind(settings);
+        });
+
         // Memory only section
         services.AddDistributedMemoryCache();
         services.AddTransient<IBlockchainConnector, LogBlockchainConnector>();
         services.AddSingleton<IEventStore, MemoryEventStore>();
         services.AddMassTransit(x =>
         {
-            x.AddConsumer<TransactionProcessor>();
-            x.AddSagaRepository<JobSaga>().InMemoryRepository();
-            x.AddSagaRepository<JobTypeSaga>().InMemoryRepository();
-            x.AddSagaRepository<JobAttemptSaga>().InMemoryRepository();
+            x.AddConsumer<TransactionProcessor>(cfg =>
+            {
+                cfg.Options<JobOptions<TransactionJob>>(options => options
```

Comment:
Is it possible to add a comment explaining why the ConcurrentJobLimit is set to 1?